ledger-keyring-mv2.js

```diff
@@ -0,0 +1,195 @@
+const { BaseLedgerKeyring } = require('./base-ledger-keyring')
+
+const CONNECTION_EVENT = 'ledger-connection-change'
+
+class LedgerKeyringMv2 extends BaseLedgerKeyring {
+  init () {
+    this.iframeLoaded = false
+    this._setupIframe()
+
+    this.currentMessageId = 0
```

Comment:
I know you didn't add this, and maybe this is beyond the scope of the PR, but `this.currentMessageId` can be pretty safely removed by replacing it with a `nanoid` in the `_sendMessage` method. Which makes messaging less bug-prone, IMO, as:

If there are two instances of the keyring that are connected to the same bridge then they'll potentially be sending out messages with the same ID (e.g., 0, 1, 2) but with different callbacks.